mesonbuild/backend/vs2010backend.py

```diff
@@ -2096,7 +2132,7 @@ def generate_lang_standard_info(self, file_args: T.Dict[str, CompilerArgs], clco
         pass
 
     # Returns if a target generates a manifest or not.
-    def get_gen_manifest(self, target):
+    def get_gen_manifest(self, target: T.Optional[build.BuildTarget]):
```

Comment:
```suggestion
    def get_gen_manifest(self, target: T.Optional[build.BuildTarget]) -> bool:
```

Might as well add all the annotations while we're here.